packages/react-app/src/pages/api/bounties/[id]/claim.ts

```diff
@@ -0,0 +1,58 @@
+import { NextApiRequest, NextApiResponse } from 'next';
+import dbConnect from '@app/utils/dbConnect';
+import { BountyClaimSchema } from '@app/models/Bounty';
+import { internalServerError, notFound } from '@app/errors';
+import * as service from '@app/services/bounty.service';
+import middlewares from '@app/middlewares';
+import { RoleRestrictions } from '@app/types/Role';
+
```

Comment:
I set claim as a separate route because I wanted to segment claimants from editors. I see these as different permissions.